docs/content/tutorials/setup.md

```diff
@@ -0,0 +1,67 @@
+# Setup
+
+After completing this tutorial you'll have your own production-ready HedgeDoc instance running.
+We will use [Docker][docker-docs] to accomplish this.
```

Comment:
I'd be **very** cautious of calling something "production-ready" without discussing backups, TLS, authentication etc.

Also, the docker-compose.yml currently does not persist the Postgres database.